deploy/handler/operator.yaml

```diff
@@ -42,81 +40,74 @@ spec:
-            - name: WEBHOOK_PORT
-              value: "54874"
+              value: "{{template "handlerPrefix" .}}nmstate"
             - name: ENABLE_PROFILER
               value: "False"
             - name: PROFILER_PORT
               value: "6060"
-            - name: NMSTATE_INSTANCE_NODE_LOCK_FILE
-              value: "/var/k8s_nmstate/handler_lock"
+          ports:
+          - containerPort: 8443
+            name: webhook-server
+            protocol: TCP
+          readinessProbe:
+            httpGet:
+              path: /readyz
+              port: webhook-server
+              scheme: HTTPS
+              httpHeaders:
+              - name: Content-Type
+                value: application/json
+            initialDelaySeconds: 10
+            periodSeconds: 10
           volumeMounts:
-          - name: dbus-socket
-            mountPath: /run/dbus/system_bus_socket
-          - name: nmstate-lock
-            mountPath: /var/k8s_nmstate
-          securityContext:
-            privileged: true
+          - name: tls-key-pair
+            readOnly: true
+            mountPath: /etc/webhook/certs
       volumes:
-      - name: dbus-socket
-        hostPath:
-          path: /run/dbus/system_bus_socket
-          type: Socket
-      - name: nmstate-lock
-        hostPath:
-          path: /var/k8s_nmstate
+        - name: tls-key-pair
+          secret:
+            secretName: {{template "handlerPrefix" .}}nmstate-webhook
 ---
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: {{template "handlerPrefix" .}}nmstate-handler-worker
+  name: {{template "handlerPrefix" .}}nmstate-handler
   namespace: {{ .HandlerNamespace }}
+  labels:
+    app: kubernetes-nmstate
+    component: kubernetes-nmstate-handler
 spec:
   selector:
     matchLabels:
-      name: {{template "handlerPrefix" .}}nmstate-handler-worker
+      name: {{template "handlerPrefix" .}}nmstate-handler
   template:
     metadata:
       labels:
         app: kubernetes-nmstate
-        name: {{template "handlerPrefix" .}}nmstate-handler-worker
+        component: kubernetes-nmstate-handler
+        name: {{template "handlerPrefix" .}}nmstate-handler
     spec:
       # Needed to force vlan filtering config with iproute commands until
       # future nmstate/NM is in place.
       # https://github.com/nmstate/nmstate/pull/440
       hostNetwork: true
       serviceAccountName: {{template "handlerPrefix" .}}nmstate-handler
-      affinity:
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-              - matchExpressions:
-                  - key: node-role.kubernetes.io/master
-                    operator: NotIn
-                    values:
-                      - ""
       nodeSelector:
         beta.kubernetes.io/arch: amd64
       {{- range $key, $value := .HandlerNodeSelector }}
       {{"  "}}{{- $key }}: {{  $value }}
       {{- end}}
+      tolerations:
```

Comment:
I can't see the master handlers DS, do we want to limit the handlers to run only on worker nodes?

Reply:
I just read in the PR description that we removing the handlers to run only on workers by default. 
What the user should do if he wants to include also the masters?
If we want to change for example the k8s control plane network using nmstate ( for example run it as tagged traffic), I guess we need nmstate to run also on masters for that case

Reply:
This PR has one type of handler daemonset that runs at master and workers, if we want to change just masters we can put in place a profile with nodeSelector `kubernetes.io/role/master=""`